protocols/isis/server/neighbor.go

```diff
@@ -230,3 +230,26 @@ func getP2PAdjTLV(tlvs []packet.TLV) *packet.P2PAdjacencyStateTLV {
 func (n *neighbor) p2pAdjTLVContainsSelf(t *packet.P2PAdjacencyStateTLV) bool {
 	return t.NeighborSystemID == n.nm.netIfa.srv.nets[0].SystemID && t.NeighborExtendedLocalCircuitID == uint32(n.nm.netIfa.devStatus.GetIndex())
 }
+
+func (n *neighbor) extendedISReachabilityNeighbor() *packet.ExtendedISReachabilityNeighbor {
```

Comment:
Do you want to tackle TODO + FIXME later?

Reply:
Leaving the TODO in. Fixed the FIXME.